obstacle_stop_planner_refine/design/class_diagram.pu

```diff
@@ -0,0 +1,43 @@
+@startuml
+
+namespace rclcpp {
+  class Node
+}
+
+namespace motion_planning {
+rclcpp.Node <|-- ObstacleStopPlannerNode
+ObstacleStopPlannerNode *-- AdaptiveCruiseController
+ObstacleStopPlannerNode *-- ObstacleStopPlannerDebugNode
+}
+
+@enduml
+
+@startuml
+
+namespace rclcpp {
+  class Node
+  class Logger
+}
+
+namespace motion_planning {
+rclcpp.Node <|-- ObstacleStopPlannerNode
+ObstacleStopPlannerNode *-- AdaptiveCruiseController
+ObstacleStopPlannerNode *-- ObstacleStopPlannerDebugNode
+ObstacleStopPlannerNode *-- ObstaclePointCloud
+ObstacleStopPlannerNode *-- VehicleInfo
+ObstaclePointCloud *-- VehicleInfo
+ObstaclePointCloud *-- rclcpp.Logger
+ObstacleStopPlannerNode *-- Trajectory
+AdaptiveCruiseController *-- rclcpp.Node
+ObstacleStopPlannerDebugNode *-- rclcpp.Node
+OneStepPolygon *-- VehicleInfo
+PointHelper *-- VehicleInfo
+
+
+vehicle_info_util.VehicleInfo <|-- VehicleInfo
+}
+namespace vehicle_info_util {
+class VehicleInfo
+}
+
+@enduml
```

Comment:
Let's set `Insert Final Newline` on in your editor!
![image](https://user-images.githubusercontent.com/31987104/109601942-1e951e80-7b63-11eb-94ec-ff4134e9c805.png)

Reply:
Thanks. I have enabled my editor settings.